{
  try {
    delegates.jobs.executeJob(jobMeta,local,remote,replayDate,safe,startCopyName,startCopyNr);
  }
 catch (  Exception e) {
    new ErrorDialog(shell,"Execute job","There was an error during job execution",e);
  }
}
